src/gui/qgsnewhttpconnection.cpp

```diff
@@ -65,13 +65,17 @@ QgsNewHttpConnection::QgsNewHttpConnection( QWidget *parent, ConnectionTypes typ
   txtName->setValidator( new QRegularExpressionValidator( QRegularExpression( "[^\\/]+" ), txtName ) );
 
   cmbDpiMode->clear();
-
   cmbDpiMode->addItem( tr( "all" ), static_cast<int>( Qgis::DpiMode::All ) );
   cmbDpiMode->addItem( tr( "off" ), static_cast<int>( Qgis::DpiMode::Off ) );
   cmbDpiMode->addItem( tr( "QGIS" ), static_cast<int>( Qgis::DpiMode::QGIS ) );
   cmbDpiMode->addItem( tr( "UMN" ), static_cast<int>( Qgis::DpiMode::UMN ) );
   cmbDpiMode->addItem( tr( "GeoServer" ), static_cast<int>( Qgis::DpiMode::GeoServer ) );
 
+  cmbTilePixelRatio->clear();
+  cmbTilePixelRatio->addItem( tr( "Unknown (not scaled)" ), static_cast<int>( Qgis::TilePixelRatio::Unknown ) );
```

Comment:
Can we call it "undefined" or "native" or "unscaled"?

Reply:
@m-kuhn , picked undefined.